public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx

```diff
@@ -1,35 +1,131 @@
+import { DataTransformerID, standardTransformersRegistry, TransformerRegistryItem } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
-import { Trans } from '@grafana/i18n';
-import { Box, Button, Stack, Text } from '@grafana/ui';
+import { t, Trans } from '@grafana/i18n';
+import { reportInteraction } from '@grafana/runtime';
+import { Box, Button, Grid, Stack, Text } from '@grafana/ui';
+import config from 'app/core/config';
+
+import { TransformationCard } from '../../../dashboard/components/TransformationsEditor/TransformationPickerNg';
 
 interface EmptyTransformationsProps {
   onShowPicker: () => void;
+  onGoToQueries?: () => void;
+  onAddTransformation?: (transformationId: string) => void;
+}
+
+const TRANSFORMATION_IDS = [
+  DataTransformerID.organize,
+  DataTransformerID.groupBy,
+  DataTransformerID.extractFields,
+  DataTransformerID.filterByValue,
+];
+
+interface SQLTransformationTile {
+  id: 'sql-transformation';
```

Comment:
I'm thinking we might want to keep it as SQL Expressions for now. Until we decide whether we want to change the name for sure.